pytest_html/plugin.py

```diff
@@ -271,9 +272,10 @@ def generate_checkbox(self):
                                            **checkbox_kwargs)
 
             def generate_summary_item(self):
-                self.summary_item = html.span('{0} {1}'.
-                                              format(self.total, self.label),
-                                              class_=self.class_html)
+                self.summary_item = [html.span('{0} {1}'.
+                                               format(self.total, self.label),
+                                               class_=self.class_html),
+                                     html.span('; ')]
```

Comment:
I added it, just because I think it looks better like that, when there isn't nothing to separate, I get confused when a report starts and other one finishes